components/ScreenerView/ScreenerView.vue

```diff
@@ -41,6 +41,9 @@
   ::v-deep
     .vue-tags-input
       max-width: initial !important
+      .ti-autocomplete
+        height: 240px
```

Comment:
Can you display only half of it at the bottom so that the user understands that it is scrollable?
Change it to max-hight for when there are fewer than 10 candidates.

`max-hight: 252px`

Reply:
Thank you for your comment!!
Based on your idea, I have made the results area resizable and also the scrollbars more obvious.
Let me know if you have any questions.
[https://drive.google.com/file/d/1Wvph2KjjJW69XD2pAP_qRr5cjU-nE-55/view?usp=sharing](url)

Reply:
Thank you for your comment!!
Based on your idea, I have made the result box resizable and the scrollbar more obvious.
Let me know if you have any questions.
https://drive.google.com/file/d/1Wvph2KjjJW69XD2pAP_qRr5cjU-nE-55/view?usp=sharing